<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_mid">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Mid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_go">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_left">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_right">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>401</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>460</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>431</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>command log</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>sensing log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>390</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>LeftSide</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>390</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Front</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>390</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>RightSide</string>
    </property>
   </widget>
   <widget class="QSlider" name="speed">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>26</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>0</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
    <property name="sliderPosition">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdTemp">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdHumi">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>430</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>온도</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>습도</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>520</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>MIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>MAX</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_go</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedGo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_right</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedRight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_back</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_left</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedLeft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_mid</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedMid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_stop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>540</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>LeftPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>LeftRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>FrontPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>696</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>FrontRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>726</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>RightPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>802</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>859</x>
     <y>562</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>RightRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>815</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>859</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setSpeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>522</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickedGo()</slot>
  <slot>clickedRight()</slot>
  <slot>clickedLeft()</slot>
  <slot>clickedMid()</slot>
  <slot>clickedBack()</slot>
  <slot>clickedFaster()</slot>
  <slot>clickedSlower()</slot>
  <slot>clickedStop()</slot>
  <slot>LeftPress()</slot>
  <slot>RightPress()</slot>
  <slot>LeftRelease()</slot>
  <slot>RightRelease()</slot>
  <slot>FrontPress()</slot>
  <slot>FrontRelease()</slot>
  <slot>upRelease()</slot>
  <slot>downRelease()</slot>
  <slot>setSpeed()</slot>
 </slots>
</ui>
